rln-research/2022-07-22-rln-relay-journey.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+name:  ""
+title: ""
+date: 2022-10-18 10:00:00
+author: sanaztaheri
+published: true
+permalink: /waku-rln-relay-journey
+categories: research
+summary: The current state of the research and development of the WAKU-RLN-RELAY project.
+image: 
+discuss: 
+_includes: [math]
+---
+
+In this post, we would like to shed light on the current state of the research and development of the WAKU-RLN-RELAY project as well as its future direction and the areas of work to which the readers of this post may wish to contribute.
+
+# WAKU-RLN-RELAY: Why and How
+
+**Why**: The WAKU-RLN-RELAY project has been started more than two years ago to protect the open p2p messaging protocol of [WAKU-RELAY](https://rfc.vac.dev/spec/11/) against spam and DDoS attacks.
+However, the use case of WAKU-RLN-RELAY is not limited to the WAKU-RELAY protocol and can be adopted by any open p2p routing system.
+It is worth mentioning that WAKU-RELAY is a minor extension of [Libp2p Gossipsub protocol](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md), hence the spam protection property of WAKU-RLN-RELAY is directly applicable to the libp2p Gossipsub protocol too. 
+In the nutshell, spam protection is achieved by limiting the messaging rate of every network participant so that no user can flood the network with a large number of messages and congest/DDoS relayers.
```

Comment:
```suggestion
In a nutshell, spam protection is achieved by limiting the messaging rate of every network participant so that no user can flood the network with a large number of messages and congest/DDoS relayers.
```